tfrecords/create_tfrecords_lowres.py

```diff
@@ -0,0 +1,156 @@
+"""
+Script to generate TFRecord shards from the Sidewalks dataset as shown in this
+blog post: https://huggingface.co/blog/fine-tune-segformer.
+
+The recommended way to obtain TFRecord shards is via an Apache Beam
+Pipeline with an execution runner like Dataflow. Example:
+https://github.com/GoogleCloudPlatform/practical-ml-vision-book/blob/master/05_create_dataset/jpeg_to_tfrecord.py.
+
+Usage:
+    python create_tfrecords --batch_size 16
+
+References:
+
+    * https://github.com/GoogleCloudPlatform/practical-ml-vision-book/blob/master/05_create_dataset/05_split_tfrecord.ipynb
+    * https://www.tensorflow.org/tutorials/images/segmentation
+"""
+
+import argparse
+import math
+import os
+from typing import Tuple
+
+import datasets
+import numpy as np
+import tensorflow as tf
+import tqdm
+from PIL import Image
+
+RESOLUTION = 256
+
+def load_sidewalks_dataset(args):
+    hf_dataset_identifier = "segments/sidewalk-semantic"
+    ds = datasets.load_dataset(hf_dataset_identifier)
+
+    ds = ds.shuffle(seed=1)
+    ds = ds["train"].train_test_split(test_size=args.split, seed=args.seed)
+    train_ds = ds["train"]
+    val_ds = ds["test"]
+
+    return train_ds, val_ds
+
+
+def normalize_img(image: np.ndarray, label: np.ndarray) -> Tuple[tf.Tensor, tf.Tensor]:
+    image = tf.cast(image, tf.float32) / 255.0
+    label -= 1
```

Comment:
Is this needed in our case? I know it's there in https://github.com/deep-diver/semantic-segmentation-ml-pipeline/blob/main/tfrecords/create_tfrecords_fullres.py too. But I guess it's not needed.

Reply:
Yeah I mean we cal eliminate that step 'cause it seems like it's redundant. I remember it actually led to a training fault?